Dockerfile.alpine

```diff
@@ -1,11 +1,10 @@
 # syntax=docker/dockerfile:experimental
 
-FROM ruby:3.0.1-alpine3.13
+FROM ruby:2.6.5-alpine3.11
 
-ENV NODE_VERSION 14.16.1-r1
-ENV APK_TOOLS_VERSION 2.12.5-r0
+ENV NODE_VERSION 12.22.1-r0
 RUN \
-  apk add --update nodejs=${NODE_VERSION} npm=${NODE_VERSION} apk-tools=${APK_TOOLS_VERSION} \
+  apk add --update nodejs=${NODE_VERSION} npm=${NODE_VERSION} apk-tools=2.10.6-r0\
```

Comment:
can use hoist APK_TOOLS_VERSION like before

Reply:
@bernardlau do you want to make that change here - good idea.